timewsync/__init__.py

```diff
@@ -184,6 +184,10 @@ def sync(configuration: Configuration) -> None:
         log.error("Unexpected error occurred during JWT generation. No changes were made.")
         return
 
+    # Active time tracking
+    if active_interval:
+        print("Time tracking is active. Stopped and restarted time tracking to prevent conflicts.", file=sys.stderr)
```

Comment:
Before the "restart" was only part of the message if we actually were able to restart the tracking. Is this change deliberate?

If not we should add a check for `started_tracking`.

Reply:
That's a good point, maybe the previous approach was better in that regard!
Will update the message a little